Makefile

```diff
@@ -146,7 +146,7 @@ staticcheck: $(SOURCES)
 # G304 reading kubernetes secret filepaths are not a file inclusions
 # G402 See https://github.com/securego/gosec/issues/551 and https://github.com/securego/gosec/issues/528
 gosec: $(SOURCES)
-	gosec -quiet -exclude="G101,G104,G304,G402" ./...
+	gosec -quiet -exclude="G101,G104,G304,G402,G114" ./...
```

Comment:
can you add what G114 is and why it's ignored as the others

Reply:
From this: https://github.com/zalando/skipper/issues/2062

I added this commit to test the build would succeed. Let's decide how to solve: https://github.com/zalando/skipper/issues/2062 in a separate PR then I can remove this commit from this PR. (Marked as `do-not-merge` to make it more clear that we should solve that first)

Reply:
Please rebase, it's fixed now

Reply:
Updated!